arduino-ide-extension/src/node/monitor-service.ts

```diff
@@ -80,6 +80,7 @@ export class MonitorService extends CoreClientAware implements Disposable {
   private readonly board: Board;
   private readonly port: Port;
   private readonly monitorID: string;
+  private streamingTextDecoder = new TextDecoder('utf8');
```

Comment:
```suggestion
  private readonly streamingTextDecoder = new TextDecoder('utf8');
```

Reply:
Just to follow convention in reality, no reason this should be modified so let's just update this.

Reply:
Makes sense to me, thanks